docsrc/firebolt.model.rst

```diff
@@ -9,7 +9,7 @@ Model.database
 
 .. automodule:: firebolt.model.database
```

Comment:
Can we rename every module header in such way:
`Model.engine_revision -> Engine revision`